src/static/css/devcade.css

```diff
@@ -46,29 +121,104 @@ a {
     vertical-align: middle;
 }
 
+.header {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background:
+      linear-gradient(rgba(0,0,0,0) 20%, var(--bg) 100%),
+      repeat top/2.5em url('../images/csh_tilted.png');
+}
+
 #bg {
-    width: 100%;
     height: min(100vh, 400px);
 }
 
 .dropdown-toggle {
     cursor: pointer;
 }
+  
+#content-wrapper {
+  margin-left: auto;
+  margin-right: auto;
+  display: inline-block;
+  display: grid;
+  grid-template-columns: 2fr 1fr;
+  grid-template-rows: auto auto auto auto;
+  grid-template-areas:
+    "a d"
+    "b d"
+    "c e";
+  grid-gap: 1em 1em;
+  max-width: 40em;
+  padding: 0 1em;
+}
+  
+.card {
+  padding: 1.875em;
+  background-color: #340526;
+  border-color: #50103D;
+  border-width: 2pt;
+  border-radius: 0.25em;
+}
```

Comment:
Looks like you just deleted my card class for the uploaded content so you could use the card class for your own cards on the homepage.  Was there an issue with my class or were we just too lazy to change the class name?